.github/workflows/PR_jira_hyperlink.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Jira Ticket Link
+
+on:
+  pull_request:
+    types: [opened, edited]
+
+jobs:
+  update-jira-link:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Extract PR description
+        id: extract-description
+        run: |
+          PR_DESCRIPTION=$(jq -r '.pull_request.body' "$GITHUB_EVENT_PATH")
```

Comment:
I'm getting
Run PR_DESCRIPTION=$(jq -r '.pull_request.body' "$GITHUB_EVENT_PATH")
  PR_DESCRIPTION=$(jq -r '.pull_request.body' "$GITHUB_EVENT_PATH")
  echo "PR_DESCRIPTION=$PR_DESCRIPTION" >> $GITHUB_ENV
  shell: /usr/bin/bash -e {0}
Error: Unable to process file command 'env' successfully.
Error: Invalid format ''